tests/utils.go

```diff
@@ -469,6 +469,64 @@ func GetSupportedCPUModels(nodes k8sv1.NodeList) []string {
 	return cpus
 }
 
+func GetSupportedCPUFeaturesAmongAllTheNodes(nodes k8sv1.NodeList) []string {
+	var featureDenyList = map[string]bool{
+		"svm": true,
+	}
+	var supportedFeatures []string
+	for label := range nodes.Items[0].Labels {
+		if strings.Contains(label, services.NFD_CPU_FEATURE_PREFIX) {
+			feature := strings.TrimPrefix(label, services.NFD_CPU_FEATURE_PREFIX)
+			if _, ok := featureDenyList[feature]; !ok {
+				supportedFeatures = append(supportedFeatures, feature)
+			}
+		}
```

Comment:
you can export this into a func and re-use [below](https://github.com/kubevirt/kubevirt/pull/7800/files#diff-847cd06c1bf6ef597ea96894f8b32ecb4d074ad1bc785b39871e11fb6ab71b3dR489-R494) - iiuc, this code is duplicate.

Reply:
Done.